src/rastervision/command/command.py

```diff
@@ -7,6 +7,15 @@ def run(self, tmp_dir):
         """Run the command."""
         pass
 
+    def set_tmp_dir(self, tmp_dir):
+        self._tmp_dir = tmp_dir
+
+    def get_tmp_dir(self):
```

Comment:
Should this check for `None` or a missing property, and then call `RVConfig.get_tmp_dir()`? If someone creates a command not through a config, and doesn't call `set_tmp_dir`, I'm not sure it should break.